src/api/facade.js

```diff
@@ -57,64 +47,52 @@ export const checkValidHash = (data: string) => {
 
 export const checkAndFacadeTransactionAsync = async (
   tx: RawTransaction,
+  bestBlockNum: number,
```

Comment:
Do we want to  continue including `bestBlockNum` as part of the `Transaction` type?

Reply:
hmm maybe we can get rid of it. I left it for backwards compatibility but I guess I can workaround the fact of not having it directly in the tx